cloudfront.tf

```diff
@@ -0,0 +1,83 @@
+locals {
+  cf_origin_id = "host"
+}
+
+resource "aws_cloudfront_distribution" "geofencing" {
+  provider = aws.iam-Perimeter
```

Comment:
We'll want to remove this provider line. We want to create cloudfront in the tnfhhm-dev account. Not the perimeter account.

Reply:
Made changes, Please reveiw